index.js

```diff
@@ -1089,6 +1222,10 @@ const createFocusTrap = function (elements, userOptions) {
 
           onUnpause?.();
 
+          if (config.isolateSubtrees) {
+            setSubtreeIsolation(true);
+          }
```

Comment:
Might be necessary to not have this tied so closely to pausing/unpausing per my previous comment, and so we might want to remove this and replace it with more _precise_ subtree isolation calls...